### DISCUSSION TITLE
GitHub Student Developer Pack and Campus Locations

### DISCUSSION THREAD

**Discussion by hyunwooed** [community/community#49172] [GitHub Education]
Hi there, I'm trying to apply for the GitHub Student Developer Pack and it appears that my requests are getting denied due to GitHub mentioning that the school I've selected "does not appear to have a campus location in my country."

To specify, I am not learning remote at the moment, but am instead enrolled in one of my institutions' "extended campuses" in South Korea, in particular the Incheon Global Campus consisting of the State University of New York (SUNY) Korea, George Mason University Korea, the University of Utah Asia Campus, and Ghent University Global Campus. I've signed up using my school email address (ending in .edu) and also submitted a ticket last month, but received no response, which is why I'm creating a discussion here.

Thank you!

**Comment by 0x5b62656e5d** [comment]
I have the same issue
GitHub support refusing to help... :<

**Comment by samus-aran** [comment]
Hi @hyunwooed 👋 

We've been able to verify this campus, we would like to ask you to re-apply. 

Let us know how you get on 👍  Hope this helps!

**Comment by hyunwooed** [comment]
Hi @samus-aran 😃
I've tried re-applying and it appears that now I'm stuck on GitHub not being able to see my location information. I've tried as it has told me, trying different devices or browsers, but to no avail - I understand that this is an issue not necessarily related to my original post, but has anyone had success fixing this issue (as it looks like others are having similar problems)?
<img width="752" alt="Screenshot 2023-03-13 at 11 17 26 AM" src="https://user-images.githubusercontent.com/126231261/224593452-64cc4d5e-b236-4650-a937-44f2c17f8082.png">
